eo-runtime/src/main/eo/org/eolang/math/random.eo

```diff
@@ -23,6 +23,48 @@
 +package org.eolang.math
 +home https://github.com/objectionary/eo
 +version 0.0.0
++alias org.eolang.math.number
 +rt jvm org.eolang:eo-runtime:0.0.0
 
-[] > random /float
+[seed] > random
+
+  # Get float value for current seed
+  # Divide seed to maximum possible value of the seed
+  div. > @
+    as-float.
+      number seed
+    as-float.
+      number
+        left.
+          number 1
+          48
+
+  # Calculates next seed.
+  # Formula is based on linear congruential pseudorandom number generator, as defined by
+  # D. H. Lehmer and described by Donald E. Knuth in The Art of Computer Programming, Volume 2,
+  # Third edition: Seminumerical Algorithms, section 3.2.1.
+  # Magic numbers are taken from Java implementation. 48 lower bits are considered.
+  # next = (seed * 25214903917 + 11) AND ((1 <LEFT SHIFT> 48) - 1)
+  and. > next-seed
+    number
+      plus.
+        11
+        times.
+          seed
+          25214903917
+    minus.
+      left.
+        number 1
+        48
+      1
+
+  # Next random
+  random > next
+    $.next-seed
+
+  # New random
+  random > pseudo
```

Comment:
@mximp it's better to keep the "surface" of an object as small as possible. Every "extra" method we introduce is a potential noise for object users. I suggest we stay with only two methods in this object: `next` and `pseudo`